concept_map.puml

```diff
@@ -26,22 +26,29 @@ cloud "GitHub Actions" as GA
 cloud Buildkite
 
 rectangle "CI integrations" as CI
+rectangle "Dart Cloud Functions" as DartCloudFunctions
 
 together {
     node Firebase {
         cloud Hosting
         database Firestore
         cloud Authentication
+        cloud "Cloud Functions" as CloudFunctions
```

Comment:
Generally speaking, the Cloud Functions and Dart Cloud Functions are the same entity for this diagram. What if we inline it or notice somehow that our functions are written in Dart?
```suggestion
        cloud "Dart Cloud Functions" as DartCloudFunctions
```
We can make this cloud green noticing that this is part of Metrics components.